### ISSUE TITLE
HoloViews search is not informative

### ISSUE THREAD

**Issue by ahuang11** [holoviz/holoviews#6150]
<img width="1046" alt="image" src="https://github.com/holoviz/holoviews/assets/15331990/17e3a159-733f-4422-ae18-1638118f0f95">

**Comment by philippjfr** [comment]
Yeah that's pretty bad, presumably because the pages contain large amounts of JSON from the Bokeh output.

**Comment by ahuang11** [comment]
Ah, yes, it's searching "default" in the output which finds `default: flex-start`.
